src/benchq/problem_ingestion/molecule_instance_generation.py

```diff
@@ -106,24 +134,91 @@ def _run_pyscf(self) -> Tuple[gto.Mole, scf.hf.SCF]:
             SCFConvergenceError: If the SCF calculation does not converge.
         """
         molecule = self.get_pyscf_molecule()
-        mean_field_object = (scf.RHF if self.multiplicity == 1 else scf.ROHF)(molecule)
+        mean_field_object = (scf.RHF if self.app_data.multiplicity == 1 else scf.ROHF)(
+            molecule
+        )
 
-        if self.scf_options is not None:
-            mean_field_object.run(**self.scf_options)
+        run_id = ""
```

Comment:
**nitpick**: is an empty string ever a valid value? If not, I'd recommend using `None` instead. This way `mypy` will tell you if you forget to set it.

Reply:
fixed in e2f44348a1f07b7640f7dd5d68ddeac3c6b616b7